packages/arb-util/protocol/timebounds.go

```diff
@@ -32,11 +32,15 @@ type TimeBounds struct {
 }
 
 func NewRandomTimeBounds() *TimeBounds {
+	upperBlockBound := common.RandBigInt()
+	lowerBlockBound := common.RandBigIntBelowBound(upperBlockBound)
```

Comment:
It seems better to generate two random BigInts and sort them.  The BelowBound approach could take a long time if you're very unlucky in the choice of the upper bound.

Reply:
Ah good call. That makes way more sense haha

Reply:
Fixed